src/gl/pointFeature.js

```diff
@@ -365,13 +390,33 @@ var gl_pointFeature = function (arg) {
   /**
    * Return the number of vertices used for each point.
    *
-   * @returns {Number}
+   * @returns {number}
    */
   this.verticesPerFeature = function () {
     var unit = pointPolygon(0, 0, 1, 1);
     return unit.length / 2;
   };
 
+  /**
+   * Set style(s) from array(s).  For each style, the array should have one
+   * value per data item.  The values are not converted or validated.  Color
+   * values should be {@link geo.geoColorObject}s.  If invalid values are given
```

Comment:
is "s" after the "}" intentional?

Reply:
Yes.  It makes the docs read 'Color values should be `geo.geoColorObject`s.', which seemed more grammatical, but I'm not committed to the 's'.

Reply:
I am fine with this. You could have said that 'Color' values should a a geo.geoColorObject.

Reply:
I reworded it a little.